web/app/Application.js

```diff
@@ -60,6 +61,8 @@ Ext.define('Traccar.Application', {
         'Notifications',
         'AllNotifications',
         'GeofenceTypes',
+        'AttributesAliases',
```

Comment:
It should be `AttributeAliases`, no plural on the first word. Same for `All...`.